cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -111,7 +111,10 @@ if(ARROW_DEFINE_OPTIONS)
 
   define_option(ARROW_POSITION_INDEPENDENT_CODE
                 "Whether to create position-independent target" ON)
-
+  
+  define_option(ARROW_USE_SCCACHE "Use sccache when compiling (if available),;\
+takes precdence over ccache if a storage backend is configured" ON)
```

Comment:
```suggestion
  define_option(ARROW_USE_SCCACHE "Use sccache when compiling (if available), \
takes precedence over ccache if a storage backend is configured" ON)
```

Reply:
Also, move this below ARROW_USE_CCACHE?

Reply:
I have moved ccache above this, the semicolon is not a typo though, it is required as a line break:
```
CMake Error at cmake_modules/DefineOptions.cmake:27 (message):
  description for ARROW_USE_SCCACHE contained a

          line 105 characters long!
          (max is 80). Split it into more lines with semicolons
```